.circleci/config.yml

```diff
@@ -318,6 +328,35 @@ jobs:
             curl -L https://github.com/aelsabbahy/goss/releases/download/v0.3.9/goss-linux-amd64 -o ./docker/tests/goss-linux-amd64
             ./gradlew --no-daemon testDocker
 
+  buildArm64Docker:
+    executor: besu_arm64_executor_med
+    steps:
+      - prepare
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: hadoLint_openjdk_11
+          command: |
+            docker run --rm -i hadolint/hadolint < docker/openjdk-11/Dockerfile
+      - run:
+          name: hadoLint_openjdk_latest
+          command: |
+            docker run --rm -i hadolint/hadolint < docker/openjdk-latest/Dockerfile
+      - run:
+          name: hadoLint_graalvm
+          command: |
+            docker run --rm -i hadolint/hadolint < docker/graalvm/Dockerfile
+      - run:
+          name: build image
+          command: |
+            ./gradlew --no-daemon distDocker
+      - run:
+          name: test image
+          command: |
+            mkdir -p docker/reports
+            curl -L https://github.com/aelsabbahy/goss/releases/download/v0.3.9/goss-linux-arm -o ./docker/tests/goss-linux-amd64
```

Comment:
is the output file `./docker/tests/goss-linux-amd64` named correctly for ARM? or should it be `./docker/tests/goss-linux-arm64`?

Reply:
This is the executable name that is used in `docker/test.sh`.  I think we could use the 'architecture' envar to name the executable.  IDK how much value there is in that over just changing the executable name to goss-linux 🤔 

It is probably better to err on the side of specificity, or at least not using the wrong arch name for the executable ;)